docroot/modules/custom/va_gov_post_api/src/Service/PostFacilityStatus.php

```diff
@@ -117,6 +109,37 @@ public function queueFacilityStatus(EntityInterface $entity) {
     return $queued_count;
   }
 
+  /**
+   * Adds facility service data to Post API queue by VAMC system.
```

Comment:
This seems misleading as it adds nothing for facility services and is just adding facility data based on a system save.
```suggestion
   * Add facility data to Post API queue by VAMC system.
```